src/content/docs/en/guides/integrations-guide/index.mdx

```diff
@@ -76,16 +76,36 @@ It's even possible to add multiple integrations at the same time!
 </PackageManagerTabs>
 
 :::note[Handling integration dependencies]
-If you see any warnings like `Cannot find package '[package-name]'` after adding an integration, your package manager may not have installed [peer dependencies](https://nodejs.org/en/blog/npm/peer-dependencies/) for you. To install these missing packages, run `npm install [package-name]`.
+If you see any warnings like `Cannot find package '[package-name]'` after adding an integration, your package manager may not have installed [peer dependencies](https://nodejs.org/en/blog/npm/peer-dependencies/) for you. To install these missing packages, run the following command:
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm install [package-name]
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm add [package-name]
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn add [package-name]
+  ```
+  </Fragment>
+</PackageManagerTabs>
 :::
 
 ### Manual Installation
 
 Astro integrations are always added through the `integrations` property in your `astro.config.mjs` file.
 
 There are three common ways to import an integration into your Astro project:
+
+<Steps>
 1. [Install an npm package integration](#installing-an-npm-package).
-2. Import your own integration from a local file inside your project.
+2. Import your own integration from a local file inside your project.
```

Comment:
I'm not sure, what symbol was there instead of a space, but VS Code marked it yellow, so I replaced it with a space...

Reply:
3 different ways are not steps. This is one of those cases where we are listing items, not dictating things that you have to do in order. In fact, if you do any one of these, you would NOT do any of the others! 😄

Reply:
Understood 🫡